src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/RepositoryCloneManager.cs

```diff
@@ -95,6 +111,60 @@ public class RepositoryCloneManager : IRepositoryCloneManager
         return path;
     }
 
+    public async Task<NativePath> PrepareClone(
```

Comment:
Can you add a method comment here indicating the purpose of this method?

Reply:
Good idea